src/modules/auth/index.js

```diff
@@ -61,7 +61,11 @@ export default class AuthModule extends Module {
 					// Remove `?` part from URL completely, if empty
 					reloadUrl = window.location.pathname + window.location.hash;
 				} else {
-					reloadUrl = window.location.pathname + '?' + qs.toString() + window.location.hash;
+					let state = qs.get("state")
+					let hash = localStorage.getItem(state);
```

Comment:
@Aringoaway 

```
let hash = localStorage.getItem(qs.get("state"));
```

and you can remove the `state` variable declaration above...

Reply:
@Pe5h4 I use this variable in two places. Here `let hash = localStorage.getItem(state);` and here `localStorage.removeItem(state);`

Reply:
@Aringoaway But you dont need to declare `state` variable in here, just use the `qs.get("state")` as in my proposal

Reply:
@Aringoaway Ok